src/ProductNFT.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.15;
+
+/**
+ * @title ProjectProductNFT
+ * @author La DAO
+ * @notice This contract contains the main functions for a Product NFT
+ */
+
+import {ERC721} from "../lib/openzeppelin-contracts/contracts/token/ERC721/ERC721.sol";
+// import {ERC721Enumerable} from "../lib/openzeppelin-contracts/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+import {ERC721URIStorage} from "../lib/openzeppelin-contracts/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+// import {ERC721Burnable} from "../lib/openzeppelin-contracts/contracts/token/ERC721/extensions/ERC721Burnable.sol";
+import {Counters} from "../lib/openzeppelin-contracts/contracts/utils/Counters.sol";
+import {Ownable} from "../lib/openzeppelin-contracts/contracts/access/Ownable.sol";
+
+contract RealEstateNFT is ERC721, ERC721URIStorage, Ownable {
```

Comment:
El nombre del contrato debe hacer match el nombre del archivo por etiqueta de [Solidity](https://docs.soliditylang.org/en/v0.8.17/style-guide.html#contract-and-library-names)

> As shown in the example below, if the contract name is Congress and the library name is Owned, then their associated filenames should be Congress.sol and Owned.sol.